deepvariant/openvino_estimator.py

```diff
@@ -88,7 +89,7 @@ def __init__(self, checkpoint_path, input_fn, model):
     tensor_shape = input_fn.tensor_shape
     freeze_graph(model, checkpoint_path, tensor_shape)
     subprocess.run([
-        mo_tf.__file__, '--input_model=model.pb', '--scale=128',
+        sys.executable, mo_tf.__file__, '--input_model=model.pb', '--scale=128',
```

Comment:
Thanks! It seems like our internal version already have this addition now.

Our latest internal __init__ looks like this:
```
  def __init__(self, checkpoint_path, input_fn, model, openvino_model_dir=''):
    tensor_shape = input_fn.tensor_shape
    openvino_model_pb = os.path.join(openvino_model_dir, 'model.pb')
    freeze_graph(model, checkpoint_path, tensor_shape, openvino_model_pb)
    mo_tf_args = [
        sys.executable, mo_tf.__file__, '--input_model=' + openvino_model_pb,
        '--scale=128', '--mean_values',
        '[{}]'.format(','.join(['128'] * tensor_shape[-1]))
    ]
    if openvino_model_dir:
      mo_tf_args.append('--output_dir=' + openvino_model_dir)
    subprocess.run(mo_tf_args, check=True)
    os.remove(openvino_model_pb)

    self.ie = IECore()
    net = self.ie.read_network(
        os.path.join(openvino_model_dir, 'model.xml'),
        os.path.join(openvino_model_dir, 'model.bin'))
    net.input_info['input'].precision = 'U8'
    self.exec_net = self.ie.load_network(
        net,
        'CPU',
        config={'CPU_THROUGHPUT_STREAMS': 'CPU_THROUGHPUT_AUTO'},
        num_requests=0)
    self.tf_sess = tf.compat.v1.Session()
    self.input_fn = input_fn
    self.model = model
    self.outputs = []
```